modules/installing-the-odh-operator-v1.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+//pv2hash: 9cbc09e0-cac2-4eb3-8f30-09e6469c5164
+
+[id='installing-the-odh-operator-v1_{context}']
+= Installing the Open Data Hub Operator (v1)
+
+.Prerequisites
+* You are using OpenShift Container Platform 4.11 or later.
+* Your OpenShift cluster has a minimum of 16 CPUs and 32GB of memory across all OpenShift worker nodes.
+* You can log in as a user with `cluster-admin` privileges.
+
+.Procedure
+. Log in to the OpenShift Container Platform as a user with `cluster-admin` privileges. For a developer installation from link:http://try.openshift.com[try.openshift.com], you can log in as the `kubeadmin` user.
```

Comment:
Suggest: "If you are performing a developer installation on link:http://try.openshift.com[try.openshift.com]"